jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -76,6 +76,12 @@ public StepsRunner(
     this.sourceFilesConfiguration = sourceFilesConfiguration;
     this.baseLayersCache = baseLayersCache;
     this.applicationLayersCache = applicationLayersCache;
+
+    ExecutorService executorService =
+        Boolean.getBoolean("jibSerialized")
```

Comment:
@briandealwis is this supposed to be `jibSerialize` or `jibSerialized`? If the latter, please update the title. Can you also update the `CHANGELOG` files?